usr/src/uts/common/fs/zfs/sys/zap.h

```diff
@@ -87,22 +88,17 @@ extern "C" {
 
 /*
  * Specifies matching criteria for ZAP lookups.
```

Comment:
On Mon, Mar 28, 2016 at 6:44 PM, Matthew Ahrens <notifications@github.com>
wrote:

> In usr/src/uts/common/fs/zfs/sys/zap.h
> <https://github.com/openzfs/openzfs/pull/77#discussion_r57646377>:
>
> > @@ -87,22 +88,17 @@ extern "C" {
> >
> >  /*
> >   * Specifies matching criteria for ZAP lookups.
>
> Is this trying to keep track of 2 things:
> 1. should we do UTF form normalization?
> 2. should we do case insensitivity (can also be thought of as "case
> normalization")?
> If so, can we represent this with a bitfield with 2 entries:
> 1. MT_NORMALIZED_FORM
> 2. MT_CASE_INSENSITIVE (or MT_NORMALIZED_CASE)
> If neither bit is set, then we are looking for an exact (strcmp()) match.
>

Yes, and probably.  Yuri, if you are willing to make that change I fully
agree with Matt that it looks like it should work.



> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openzfs/openzfs/pull/77/files/713d9ff5d7adc49d9e5858a05121641625b6f8d3#r57646377>
>